content/docs/typechecking-with-proptypes.md

```diff
@@ -30,6 +30,8 @@ Greeting.propTypes = {
 };
 ```
 
+In this example, we are using a class component, but the same functionality could also be applied to the so-called stateless or functional components.
```

Comment:
```suggestion
In this example, we are using a class component, but the same functionality could also be applied to function components, or components returned from [`React.memo`](https://reactjs.org/docs/react-api.html#reactmemo) or [`React.forwardRef`](https://reactjs.org/docs/react-api.html#reactforwardref).
```

We no longer use the term functional or stateless components (misleading due to hooks). Added memo and forwardRef as other examples.